models/prices/bnb/prices_bnb_tokens.sql

```diff
@@ -264,12 +264,13 @@ FROM
     ('usdv-verified-usd', 'bnb', 'USDV', 0x323665443cef804a3b5206103304bd4872ea4253,6),
     -- ('defi-defi-eth', 'bnb', 'DEFI', 0x6d106c0b8d2f47c5465bdbd58d1be253762cbbc1,18), --DEFI does not match CoinPaprika source: DEFI  
     --('dofn-openfabric', 'bnb', 'OFN', 0x8899ec96ed8c96b5c86c23c3f069c3def75b6d97,18),--not found in api 
-    ('xpe-xpense', 'bnb', 'XPE', 0x88691f292b76bf4d2caa5678a54515fae77c33af,18)
+    ('xpe-xpense', 'bnb', 'XPE', 0x88691f292b76bf4d2caa5678a54515fae77c33af,18),
+    ('ai-sleepless-ai', 'bnb', 'AI', 0xBDA011D7F8EC00F66C1923B049B94c67d148d8b2,18)
```

Comment:
are you certain this one doesn't already exist?
the CI logs show that the test is failing for unique address value in this spell

Reply:
I checked on Dune for the sleepless AI. and it's not on Dune yet. So I added it.

Reply:
look at line 263 in this file. the address already exists

Reply:
That's right, someone has already submitted it. However, why is the price data still wrong in Dune. But, never mind, I guess it's not possible to add the 'bnb' section anymore. 
Thank you for your help. Sorry for the trouble. Maybe the prices_ethereum_tokens.sql can be merged. Because I think there is no problem.
Thank you.

Reply:
as for the price on that bnb token, if it was added recently, it can take a little time to propogate through to `prices.usd`

Reply:
okay, all good on ethereum